git/filter_process_scanner.go

```diff
@@ -172,14 +173,14 @@ func (o *FilterProcessScanner) Err() error { return o.err }
 // will read the body of the request. Since the body is _not_ offset, one
 // request should be read in its entirety before consuming the next request.
 func (o *FilterProcessScanner) readRequest() (*Request, error) {
-	requestList, err := o.pl.readPacketList()
+	requestList, err := o.pl.ReadPacketList()
 	if err != nil {
 		return nil, err
 	}
 
 	req := &Request{
 		Header:  make(map[string]string),
-		Payload: &pktlineReader{pl: o.pl},
+		Payload: pktline.NewPktlineReaderFromPktline(o.pl, 32768),
```

Comment:
Yes, that's correct.  It probably makes sense to increase this to 64k, since I'm pretty sure that Git will send larger packets.  Scutiger's `git-lfs-transfer` will not (it prefers 32k chunks because they're the largest power of 2 supported in a pkt-line), but Git almost certainly will.